torch/_export/serde/serialize.py

```diff
@@ -345,6 +345,7 @@ def __init__(
         self.graph_signature = graph_signature
         self.call_spec = call_spec
         self.module_call_graph = module_call_graph
+        self.state_dict: Dict[str, torch.Tensor] = {}
```

Comment:
I don't think state_dict is the right place to smuggle this. The invariant we wanna keep is:
```
ep = export(f)
f.state_dict() == serialize(ep).state_dict
```
We could just model this as "constants" which is a separate concept and put them into a separate place. By inserting the extra tensor here we're redefining the state_dict from serializer and serializer shouldn't be in charge of IR spec.